js/tests/unit/carousel.spec.js

```diff
@@ -1079,7 +1095,326 @@ describe('Carousel', () => {
+        '    <li data-bs-target="#myCarousel" data-bs-slide-to="1"></li>',
+        '    <li data-bs-target="#myCarousel" data-bs-slide-to="2"></li>',
+        '  </ol>',
+        '  <div class="carousel-inner">',
+        '    <div class="carousel-item active">item 1</div>',
+        '    <div class="carousel-item">item 2</div>',
+        '    <div class="carousel-item">item 3</div>',
+        '  </div>',
+        '</div>',
+        '<button type="button" class="btn btn-icon btn-secondary pause" data-bs-control="play-button" data-bs-target="#myCarousel" data-bs-play-text="" data-bs-pause-text="" title="">',
+        '  <span class="visually-hidden"></span>',
+        '</button>'
+      ].join('')
+
+      const buttonPlayPauselEl = fixtureEl.querySelector('.pause')
+      const buttonPlayPauselContentEl = fixtureEl.querySelector('span.visually-hidden')
+
+      buttonPlayPauselEl.click()
+
+      expect(buttonPlayPauselEl.getAttribute('title')).toEqual('Play Carousel')
+      expect(buttonPlayPauselContentEl.innerHTML).toEqual('Play Carousel')
+
+      buttonPlayPauselEl.click()
+
+      expect(buttonPlayPauselEl.getAttribute('title')).toEqual('Pause Carousel')
+      expect(buttonPlayPauselContentEl.innerHTML).toEqual('Pause Carousel')
+    })
+
+    it('should replace button text and attributes by default texts after a click on the pause button when data-bs-*-text attributes are missing', () => {
+      fixtureEl.innerHTML = [
+        '<div id="myCarousel" class="carousel slide">',
+        '  <ol class="carousel-indicators">',
+        '    <li data-bs-target="#myCarousel" data-bs-slide-to="0" class="active"></li>',
+        '    <li data-bs-target="#myCarousel" data-bs-slide-to="1"></li>',
+        '    <li data-bs-target="#myCarousel" data-bs-slide-to="2"></li>',
+        '  </ol>',
+        '  <div class="carousel-inner">',
+        '    <div class="carousel-item active">item 1</div>',
+        '    <div class="carousel-item">item 2</div>',
+        '    <div class="carousel-item">item 3</div>',
+        '  </div>',
+        '</div>',
+        '<button type="button" class="btn btn-icon btn-secondary pause" data-bs-control="play-button" data-bs-target="#myCarousel" title="">',
+        '  <span class="visually-hidden"></span>',
+        '</button>'
+      ].join('')
+
+      const buttonPlayPauselEl = fixtureEl.querySelector('.pause')
+      const buttonPlayPauselContentEl = fixtureEl.querySelector('span.visually-hidden')
+
+      buttonPlayPauselEl.click()
+
+      expect(buttonPlayPauselEl.getAttribute('title')).toEqual('Play Carousel')
+      expect(buttonPlayPauselContentEl.innerHTML).toEqual('Play Carousel')
+
+      buttonPlayPauselEl.click()
+
+      expect(buttonPlayPauselEl.getAttribute('title')).toEqual('Pause Carousel')
+      expect(buttonPlayPauselContentEl.innerHTML).toEqual('Pause Carousel')
+    })
+    // End mod
+
+    it('should call cycle if the carousel have carousel-item-next and carousel-item-prev class', () => {
```

Comment:
Isn't it `it('should call cycle if the carousel have carousel-item-next or carousel-item-prev class, cause is sliding', () => {` in Bootstrap?